packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -480,27 +479,17 @@ export class PerformanceEvent {
     }
 }
 
-// Note - these Telemetry types should move to common-definitions package
-
 /**
  * Broad classifications to be applied to individual properties as they're prepared to be logged to telemetry.
  * Please do not modify existing entries for backwards compatibility.
  */
-export enum TelemetryDataTag {
+ export enum TelemetryDataTag {
     /** Data containing terms from code packages that may have been dynamically loaded */
     PackageData = "PackageData",
     /** Personal data of a variety of classifications that pertains to the user */
     UserData = "UserData",
 }
 
-/**
- * A property to be logged to telemetry containing both the value and the tag
- */
-export interface ITaggedTelemetryPropertyType {
-    value: TelemetryEventPropertyType,
-    tag: TelemetryDataTag
-}
-
 /**
  * Property bag containing a mix of value literals and wrapped values along with a tag
  */
```

Comment:
`ITaggableTelemetryProperties` shouldn't be necessary anymore - replace with ITelemetryProperties wherever this is used

Reply:
fixed